heathskin/game_state.py

```diff
@@ -78,6 +86,13 @@ def _get_heroes_from_entities(self):
     def _is_player_first(self):
         return self.entities.get('36').get_tag('ZONE') == 'OPPOSING PLAY (Hero)'
 
+    def _get_player_healths(self):
+        player1_hero, player2_hero = self._get_heroes_from_entities()
+        player1_health = (30 - player1_hero.get_tag('DAMAGE', 0))
```

Comment:
As is, this correctly tracks the hero health, factoring in armor and healing. In the future, I would like to implement these stats separately.